### PR TITLE
Update links.md

### PR DESCRIPTION
**Pull request by poornimanayar**
Hi,

I spoke about GraphQL. in .NET Core in April which was very well received. I thought it might be useful for anyone who wants to start off with GraphQL.NET

Poornima

### CODE REVIEW

File: docs2/site/docs/guides/links.md [hunk 1]
```diff
@@ -6,3 +6,5 @@
...
 * [Generic data loaders for Entity Framework in GraphQL](http://corstianboerman.com/posts/2019-02-12/generic-data-loaders-for-entity-framework-in-graphql.html) by [Corstian Boerman](https://twitter.com/corstianboerman) - Article describing the implementation of a generic data-loader using Entity Framework. With this approach most of the complexities related to the data-loader are abstracted away, and can easily be used from resolvers.
 * [ASP.NET Core 2.1 and GraphQL; Adding JWT Bearer validation to subscriptions](http://corstianboerman.com/posts/2018-09-10/asp-net-core-2-1-and-graphql-adding-jwt-bearer-validation-to-subscriptions.html) by [Corstian Boerman](https://twitter.com/corstianboerman) - Authorizing subscriptions works a bit different from authorizing queries and mutations. This article describes in a technical way how to use JWT based authorization on subscriptions.
+* [GraphQL in .NET Core] (https://www.youtube.com/watch?v=M6L20FMtbsA) - Talk by Poornima Nayar at the Dot Net North Meetup about creating GraphQL API in .NET Core and consuming it in a Blazor WASM App.
```
Comment (Shane32) [inline docs2/site/docs/guides/links.md:9]:
```suggestion
* [GraphQL in .NET Core](https://www.youtube.com/watch?v=M6L20FMtbsA) - Talk by Poornima Nayar at the Dot Net North Meetup about creating GraphQL API in .NET Core and consuming it in a Blazor WASM App.
```

File: docs2/site/docs/guides/links.md [hunk 2]
```diff
@@ -6,3 +6,5 @@
...
+* [Example Repo] - GitHub repo for Poornima Nayar's talk at the Dot Net North Meetup. The repo contains a sample query, mutation and subscription and example of how that is consumed in a Blazor WASM App.
```
Comment (Shane32) [inline docs2/site/docs/guides/links.md:10]: The GitHub url for the example repo is missing

**Comment by poornimanayar** [comment]
Sorry about the silly mistake. The repo is now in!

**Comment by SimonCropp** [comment]
@Shane32 ready for a merge?

**Review by Shane32 — APPROVED** [review]

**Comment by Shane32** [comment]
@SimonCropp if it looks good to you also, please approve it and I'll merge it - thanks!

**Comment by SimonCropp** [comment]
i dont have approve permissions

**Pull request merged**